security/cert_manager_operator/cert-manager-operator-install.adoc

```diff
@@ -11,6 +11,11 @@ The {cert-manager-operator} is not installed in {product-title} by default. You
 :FeatureName: The {cert-manager-operator}
 include::snippets/technology-preview.adoc[leveloffset=+1]
 
+[WARNING]
+====
+This Technology Preview version of the {cert-manager-operator} uses the upstream cert-manager version 1.7, which is no longer supported upstream. A future version of {product-title} is expected to release using a supported upstream cert-manager version. For more information, see link:https://cert-manager.io/docs/installation/supported-releases/#support-policy[Supported Releases] in the upstream cert-manager documentation.
```

Comment:
I don't think we need to say `uses the upstream cert-manager version 1.7, which is no longer supported upstream.` or `A future version of {product-title} is expected to release using a supported upstream cert-manager version. For more information, see link:https://cert-manager.io/docs/installation/supported-releases/#support-policy[Supported Releases] in the upstream cert-manager documentation.`

What Red Hat 'ships' or delivers is 'the product we ship/deliver'. Its ours we own it; we life cycle it; we define the terms by what we update it; maintain it; etc. 

In short even though we use a version that 'upstream considers EOL' we can still deliver this; support it (independently of the upstream); we do this with other software offerings all the time. IE: PostgreSQL